iron-overlay-behavior.html

```diff
@@ -15,7 +15,7 @@
 <link rel="import" href="iron-overlay-manager.html">
 
 <script>
-
+'use strict';
```

Comment:
Can you wrap this in a IIFE? For the most part it doesn't matter, but in some cases not doing this will mess up script concatenation.